tests/step-109.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+# Test:
+#   Run an install that deletes/backupds existing detected LFS hooks in existing repos
+
+if echo "$EXTRA_INSTALL_ARGS" | grep -q "use-core-hookspath"; then
+    echo "Using core.hooksPath"
+    exit 249
+fi
+
+mkdir -p /tmp/test109.1/.githooks/pre-commit &&
+    cd /tmp/test109.1 &&
+    echo 'echo "In-repo" >> /tmp/test-109.out' >.githooks/pre-commit/test &&
+    git init && mkdir -p .git/hooks &&
+    echo 'echo "Previous1" >> /tmp/test-109.out ; # git lfs arg1 arg2' >.git/hooks/pre-commit &&
+    chmod +x .git/hooks/pre-commit ||
+    exit 1
+
+mkdir -p /tmp/test109.2/.githooks/pre-commit &&
+    cd /tmp/test109.2 && git init && mkdir -p .git/hooks &&
+    echo 'echo "Previous2" >> /tmp/test-109.out ; # git lfs arg1 arg2' >.git/hooks/pre-commit &&
+    chmod +x .git/hooks/pre-commit ||
+    exit 1
+
+mkdir -p /tmp/test109.3/.githooks/pre-commit &&
+    cd /tmp/test109.3 && git init && mkdir -p .git/hooks &&
+    echo 'echo "Previous3" >> /tmp/test-109.out ; # git lfs arg1 arg2' >.git/hooks/pre-commit &&
```

Comment:
Can you make one of these `git-lfs` to text that code path too?

Reply:
made three different versions